05.wc/wc.rb

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+PRINTING_WIDTH = 8
+
+def main
+  options = ARGV.getopts('lwc')
+  ARGV.none? ? print_pipelined_input(options) : print_specified_files(ARGV, options)
+end
+
+def print_pipelined_input(options)
+  input_data_lines = $stdin.readlines
```

Comment:
標準入力から取得したデータなので、

```suggestion
  stdin_lines = $stdin.readlines
```

とかでもいいかもしれません